rpc/websocket.go

```diff
@@ -44,11 +48,36 @@ func (s *Server) WebsocketHandler(allowedOrigins []string) http.Handler {
 			utils.DebugLog("WebSocket upgrade failed", "err", err)
 			return
 		}
+
+		conns = append(conns, conn)
+		utils.DebugLog("Server Codec New...", len(conns))
 		codec := newWebsocketCodec(conn)
 		s.ServeCodec(codec, 0)
 	})
 }
 
+func BroadcastWs(method string, paramsIn ...interface{}) error {
```

Comment:
not sure why you need broadcast to ws instead of using subscription. broadcast to ws doesn't looks like a good idea